tests/unit/commands/samconfig/test_samconfig.py

```diff
@@ -0,0 +1,189 @@
+"""
+Tests whether SAM Config is being read by all CLI commands
+"""
+
+import os
+import shutil
+import tempfile
+from pathlib import Path
+from contextlib import contextmanager
+from samcli.lib.config.samconfig import SamConfig, DEFAULT_ENV
+
+from click.testing import CliRunner
+
+from unittest import TestCase
+from mock import patch
+import logging
+
+LOG = logging.getLogger()
+logging.basicConfig()
+
+config = """
```

Comment:
Lets mark this as a comment maybe, I dont think we use this config, but can be a good way to showcase an example config.